### ISSUE TITLE
docs: 希望在文档里标注下运行需要的最小配置参数

### ISSUE THREAD

**Issue by Huansheng1** [apitable/apitable#1732]
## What documentation do you want?
希望能够在 快速启动 和 开发配置 的地方能明确标注下 运行所需的最低配置，我在阿里云运行开发的步骤，直接让整个服务器卡死宕机了

**Comment by mr-kelly** [comment]
Good point. Running APITable (especially with all its microservices) can be resource-intensive. We generally recommend at least **4 Cores and 8GB RAM** for a smooth development/deployment experience. We will update the documentation to make the minimum hardware requirements clearer.

---

很好的建议。由于 APITable 包含多个微服务，运行确实比较消耗资源。通常我们建议至少需要 **4核 CPU 和 8GB 内存** 才能保证流畅的部署和开发体验。我们会更新文档，将最低硬件配置要求标注得更明显。
